eng/common/internal/Directory.Build.targets

```diff
@@ -0,0 +1,3 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```

Comment:
Wasn't this file deleted?

Reply:
Yeah I was surprised too. It somehow smuggled itself into the update.

Reply:
cc: @jcagme @mmitche -- this file was deleted in a PR which is contained in this update.

Is this a bug? https://github.com/dotnet/arcade/pull/2024

Reply:
Or could it be that @ViktorHofer used an old version of darc CLI?

Reply:
That is very likely.

Reply:
I think we should update the PR to not bring an unnecessary file.

Reply:
Updated to the latest version of darc but the file was still not flagged as deleted. I removed it manually.